README.md

```diff
@@ -11,17 +11,21 @@
     # # # https://github.com/U-S-NRL-Marine-Meteorology-Division/
 
 
-Basic GeoIPS Plugin Template 
-=============================
+Data Fusion GeoIPS Plugin Template 
+============================================
+
+This template repository contains everything necessary to create a fully compatible GeoIPS Plugin Package,
+with a focus on creating products using the data_fusion package. Currently, only the "layered" image
+functionality is templated, but in the future we will add templates for pure data fusion products (ie,
+algorithms that take in more than one data type), stitched imagery products, as well as layered imagery.
 
-This template repository contains everything necessary to create a fully compatible GeoIPS Plugin Package.
 Each file within this repository contains appropriate modification instructions.
 
 Follow the 
-[step by step instructions](https://github.com/NRLMMD-GEOIPS/template_basic_plugin/blob/dev/docs/template_instructions.rst)
+[step by step instructions](https://github.com/NRLMMD-GEOIPS/template_basic_plugin/blob/main/docs/template_instructions.rst)
```

Comment:
@mindyls  Should we also use relative paths here?

Reply:
@mindyls, Nevermind! This points to another repo, so this cannot be relative. Marking this comment as resolved.